pandas/tests/io/formats/test_to_html.py

```diff
@@ -15,6 +15,8 @@
 
 import pandas.io.formats.format as fmt
 
+lorum_ipsum = "Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum."  # noqa
```

Comment:
can you be specificerror when using noqa

Reply:
if your happy using that text, i'll break the string up over several lines instead of using the # noqa directive

Reply:
yes that would be better actually

Reply:
or make it a fixture, or use an existing fixture. i'm open to suggestions

Reply:
oh a fixture would be good

Reply:
top level conftest.py? session scope is ok for immutable string